lib/beaker-answers/versions/version20162.rb

```diff
@@ -9,5 +9,24 @@ class Version20162 < Version20161
     def self.pe_version_matcher
       /\A2016\.2/
     end
+
+    def generate_answers
+      the_answers = super
+
+      return the_answers if @options[:masterless]
+      console = only_host_with_role(@hosts, 'dashboard')
```

Comment:
I think this line isn't necessary, since you have already called `super`; you should be able to delete this line and just use the `dashboard` that is supplied [here](https://github.com/puppetlabs/beaker-answers/blob/master/lib/beaker-answers/versions/version20.rb#L114).